elyra/cli/pipeline_app.py

```diff
@@ -149,6 +151,42 @@ def _preprocess_pipeline(pipeline_path: str, runtime: str, runtime_config: str)
     return pipeline_definition
 
 
+def _print_issues(issues):
+    # print validation issues
+    for issue in issues:
+        if issue.get('severity') == ValidationSeverity.Error:
+            click.echo(
+                f'- (Fatal error on node \'{issue["data"]["nodeName"]}\' property \'{issue["data"]["propertyName"]}\') '
```

Comment:
Per #217 I agree that deciphering what fields are in `data` is not friendly.  However, I would approach this as just JSON and print it in a JSON-friendly manner and not try to figure out the "pieces".  If we did that, it might be better to change the name `data` to `info` as the idea is that that subdictionary provides information pertaining to the severity and, in many cases, is empty.  (I see a change necessary in `add_message()`)

The response printer could then ignore empty (Falsey) top-level fields where `data` (`info`) can be one.